File: backend/src/main/java/com/demo/student_management/dto/baocao/BaoCaoHocKyResponse.java
package com.demo.student_management.dto.baocao;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class BaoCaoHocKyResponse {
    private String idLop;
    private String tenLop;
    private String idHocKy;
    private Integer tongHocSinh;
    private Integer soLuongDat;
    private Double tyLeDat;
}

File: frontend/StudentManagement.Shared/Dtos/BaoCao/BaoCaoHocKyResponse.cs
namespace StudentManagement.Shared.Dtos.BaoCao;

public class BaoCaoHocKyResponse
{
    public string IdLop { get; set; } = string.Empty;
    public string TenLop { get; set; } = string.Empty;
    public string IdHocKy { get; set; } = string.Empty;
    public int TongHocSinh { get; set; }
    public int SoLuongDat { get; set; }
    public double TyLeDat { get; set; }
}
